packages/devtools_flutter/lib/src/config.dart

```diff
@@ -10,28 +11,111 @@ import 'screen.dart';
 
 /// Top-level configuration for the app.
 @immutable
-class Config {
-  /// The routes that the app exposes.
-  Map<String, WidgetBuilder> get routes {
-    final routeToBuilder = <String, WidgetBuilder>{};
-    for (var key in _routeToTabs.keys) {
-      routeToBuilder[key] =
-          (BuildContext context) => DevToolsScaffold(tabs: _routeToTabs[key]);
+class DevToolsApp extends StatefulWidget {
+  @override
+  State<DevToolsApp> createState() => DevToolsAppState();
+}
+
+/// The DevTools app's state.
```

Comment:
nit: this comment just restates the class name. Instead indicate what the DevToolsAppState means.

Reply:
Done.